### ISSUE TITLE
React 15

### ISSUE THREAD

**Issue by andykent** [tonyhb/redux-ui#31]
There is a dependency on `react ^0.14.3` this causes issues when trying to use alongside react 15. 

I'd suggest a move to a peerDependency and including react 15. Many other similar react ecosystem libraries seem to have gone with something like...

```
"peerDependencies": {
  "react": "^0.14.3 || ^15.0.1",
  "react-dom": "^0.14.3 || ^15.0.1"
}
```

Hopefully you are happy to make this change, this library is the last thing holding me back from a React 15 upgrade.

**Comment by tonyhb** [comment]
Thanks, will do

**Comment by tonyhb** [comment]
Fixed in 0.0.14, thanks!
